remoting/src/main/java/org/apache/rocketmq/remoting/transport/mqtt/RemotingCommand2MqttMessageHandler.java

```diff
@@ -38,6 +42,17 @@
     @Override
     protected void encode(ChannelHandlerContext ctx, RemotingCommand msg, List<Object> out)
             throws Exception {
-
+        if (!(msg instanceof RemotingCommand)) {
+            return;
+        }
+        MqttMessage mqttMessage = null;
+        MqttHeader mqttHeader = (MqttHeader) msg.decodeCommandCustomHeader(MqttHeader.class);
+        Message2MessageEncodeDecode message2MessageEncodeDecode = EncodeDecodeDispatcher
+                .getEncodeDecodeDispatcher().get(
+                        MqttMessageType.valueOf(mqttHeader.getMessageType()));
+        if (message2MessageEncodeDecode != null) {
+            mqttMessage = message2MessageEncodeDecode.encode(msg);
+        }
+        out.add(mqttMessage);
```

Comment:
Here,if the return value of message2MessageEncodeDecode is null,then the out object list will be add into a null value.You may add some judgement logic here.